components/CreateOrganizationForm.js

```diff
@@ -1,289 +1,390 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import { get } from 'lodash';
+import { Field, Form, Formik } from 'formik';
+import { trim } from 'lodash';
+import { withRouter } from 'next/router';
 import { defineMessages, FormattedMessage, injectIntl } from 'react-intl';
+import slugify from 'slugify';
 
-import { defaultBackgroundImage } from '../lib/constants/collectives';
-
-import InputField from './InputField';
+import { BackButton, messages } from './create-collective/CreateCollectiveForm';
```

Comment:
@Betree is there anything against importing messages from another component or would we redefine them here?

Reply:
It's better to copy-paste them, especially since we're already have a `defineMessages` in this file.

`lib/i18n` is used to group strings when needed, but it's ok to duplicate here since there are very few of them.

Reply:
fixed